fixtures/import-npm/packages/import-example/package.json

```diff
@@ -15,7 +15,7 @@
 	},
 	"devDependencies": {
 		"@cloudflare/workers-tsconfig": "../../../../packages/workers-tsconfig",
-		"undici": "^5.28.4",
```

Comment:
Should we move this to the catalog?
Oh actually it looks this it already is there 45 times out of the 46 usages